<launch>

    <node pkg="turn_htcbot" name="auto_start_node" type="auto_start.py" output="screen">
    </node>

</launch>